barcode/barcode_utils.py

```diff
@@ -2,8 +2,8 @@
 
 import base64
 import struct
-import skip32
-import code128
+import barcode.skip32
+import barcode.code128
 import os
```

Comment:
oh actually, the puppet deploy does indeed hit each plugins' requirements.txt.  skip32 (definitely) and code128 (maybe), as far as a I know, aren't python packages, they're just piles of code that are included here as part of the barcode module.

I guess  we could break these out into separate python packages if we wanted to do it that way.